### ISSUE TITLE
Example for LMGradientAttribution is missing.

### ISSUE THREAD

**Issue by saxenarohit** [pytorch/captum#1237]
## 📚 Documentation

This is in reference to the tutorial page below.
https://captum.ai/tutorials/Llama2_LLM_Attribution

I could not find the example for LLMGradientAttribution for LLAMA2.

Any help on this will be appreciated.

Thanks

**Comment by aobo-y** [comment]
@saxenarohit thanks for reminding us. We will add it soon

**Comment by Dongximing** [comment]
Hi aobo, if you are busy, could you tell me which model layer need I take as a parameters in the LayerIntegratedGradients ?

**Comment by aobo-y** [comment]
hi @Dongximing , it should be the embedding layer of your model. As a token is discrete, its backpropagate gradient stop at its embedding. For Llama2, it would something like the following

```
emb_layer = model.get_submodule("model.embed_tokens")
lig = LayerIntegratedGradients(model, emb_layer)
```

**Comment by Dongximing** [comment]
Thanks,  I saw the result, and analysis the code, the final results are computed on log_softmax. and is that means if a contribution in this way, [-10,20,-20](Assuming). the token_1 and token_2 are both important? or we need "abs()" to eval the important of tokens?

**Comment by qingyuanxingsi** [comment]
Any tutorial update?

**Comment by craymichael** [comment]
Hi, we have finally added this in https://github.com/pytorch/captum/commit/e01c07b741be53bbf77d425725088cc0dd430005. Thanks again for bringing this to our attention!
